sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClient.java

```diff
@@ -422,7 +422,7 @@ public void deleteRule(String topicName, String subscriptionName, String ruleNam
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public void deleteSubscription(String topicName, String subscriptionName) {
-        asyncClient.deleteSubscription(topicName, subscriptionName);
+        asyncClient.deleteSubscription(topicName, subscriptionName).block();
```

Comment:
Update javadoc:
"Deletes a subscription matching the {@code subscriptionName} in topic {@code topicName}."